### ISSUE TITLE
Add a base url.

### ISSUE THREAD

**Issue by prithvitewatia** [iakhil/wordatlas#11]
Currently while running the server, when we visit localhost:8000 the we get error as  we have no base url. Thus we require a base url that would redirect the user to login page if the user is not logged in and to home view if the user is logged in

**Comment by prithvitewatia** [comment]
In fact we can simplify the home view url to '' and the game_over url can also be simplified to '/game_over'

**Comment by iakhil** [comment]
Yes, we can redirect the base URL to the login page. Also, please look into the authentication issue that redirects the user to the login page after logging in with the right username and password.

**Comment by iakhil** [comment]
Yes, we can redirect the base URL to the login page. Also, please look into the authentication issue that redirects the user to the login page after logging in with the right username and password.
